### ISSUE TITLE
Update Pocket to Match Mocks

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-tv#2280]
### Vision statement / What / Requirements

The current implementation doesn't match the mocks, which were created to provide the best experience for users and match the overall channels design. This ticket is to update the Pocket channel to match those [mocks](https://brampitoyo.github.io/people/bpitoyo/firefox-for-fire-tv/new-homescreen/i04/#artboard1)

### Impact

* Improved User Experience
* Consistency between channels

### Acceptance criteria

* The following updates have been made to the Pocket channel to match the mocks:
- [x] Full-tile image for each tile
- [ ] Update Pinned tiles to use [new full-tile svgs](https://drive.google.com/drive/folders/15IFUqAoukdbAC-UEIKxD-nvrR-JeoFaf)
- [x] Highlight focus ring
- [x] 3-line wrapped title (rather than a single title space)
- [x] Spacing between tile and titles
- [ ] [blocked] Video length (this isn't currently included in the Pocket API, but would be good to add! Filed #2299)

**Comment by liuche** [comment]
Moving this into the sprint bc the mocks exist and we need it for shipping Pocket.

**Comment by liuche** [comment]
Also, these mocks look different from the original ones that we had - there, we had a single title space that spanned a longer width to accommodate the longer titles of Pocket Stories. In this version of the mock each tile has its own title underneath.

We've already implemented the single long title, so I'll have to undo that here.

**Comment by liuche** [comment]
@brampitoyo Do you think you could get us the SVG assets for the bundled Pinned Tiles? Otherwise the Pinned Tiles with the current icons would look very boxy (since the assets aren't full-tile size).
* Youtube
* Google Video Search
* IMDB
* Rotten Tomatoes
* Metacritic
* Fandango
* Hollywood Reporter
* Flickr
* Instagram
* Pinterest

**Comment by liuche** [comment]
![Screenshot_1558490606](https://user-images.githubusercontent.com/1082253/58142234-22a8b400-7bfb-11e9-8e4d-23753d267a51.png)


Things that I didn't do: 
- [ ] [**MVP**] Need to add the new Pinned Tile images so the tiles are full-size - @bram is making these as pngs, so we should be able to just swap them in. I filed #2306 to handle this.
- [ ] Settings currently isn't a "default channel", so its scrolling behavior looks pretty different. HOWEVER all the rest of the tiles are now `ListRowView` (same as Settings) so it's probably not TOO much more work to unify the code now, so the scrolling and everything is consistent 🤔 
- [ ] Didn't do the double outline on the tile focus ring, ran out of time (but this one matches the Settings focus ring)
- [ ] We have code from a previous spec to scroll the tiles so the selection is always on the left-most tile. I didn't remove that (but I also think it looks good!)

**Comment by brampitoyo** [comment]
@liuche wrote:

> Need to add the new Pinned Tile images so the tiles are full-size - @bram is making these as pngs, so we should be able to just swap them in […]

All site logo tiles in PNG format can be seen here: https://drive.google.com/drive/folders/16f4ieORIlkYyApAnb9-1d8ev2yyQ15kT?usp=sharing

They’re grouped by categories as Content Strategy has recommended.

I’ll upload SVG logo tiles (which isn’t just the logo only, but the logo + the background) tomorrow or in a couple of days.

**Comment by athomasmoz** [comment]
@liuche @brampitoyo The intent of this story was to match Pocket to the mocks, so we can get it out the door. I see that the scope has grown to include pinned tile logos, which isn't necessary to fit the vision. Since we still need to get approval on the logos we're planning to use, I'd like to remove that from this story so we're not blocked on getting the Pocket updates out. I'd also like to cross out "Video length" from this story, since it's tracked in another story that's blocked. 

Any concerns with me taking those out?

**Comment by liuche** [comment]
@athomasmoz what happened was, since Pinned Tiles is also using the same channel code as Pocket Stories, it was actually easier to make them both use the same format.

However, we came up with a contingency plan which is:
* Code freeze with white backgrounds on existing Pinned Tiles (sizes are weird)
* Get properly sized Pinned Tiles icons with white backgrounds from Bram as pngs, drop them into 3.10 release
